instances/bodySubstance/blood.jsonld

```diff
@@ -2,8 +2,8 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/blood",
-  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+  "@id": "https://openminds.ebrains.eu/instances/bodySubstance/blood",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/BodySubstance",
   "definition":  "'Blood' is a body fluid. It runs through the circulatory system of vertebrates and delivers necessary substances such as nutrients and oxygen to the cells, and transports metabolic waste products away from those same cells.",
   "description": "A bodily fluid that is composed of blood plasma and erythrocytes (blood cells).",
```

Comment:
"Blood is a body fluid composed of four main components: plasma, erythrocytes (red blood cells), leukocytes (white blood cells) and thrombocytes (platelets). Its function in the circulatory system is to deliver nutrients and oxygen to cells and transport substances such as metabolic waste products away from the cells."

Reply:
Updated in 917ed9b0c841a8517b62b18aa4646b12b3099338